src/blue-theme.scss

```diff
@@ -43,3 +43,16 @@ $primary: $democracy-blue;
 [class*="toast"] {
     margin-bottom: 0 !important;
 }
+
+[class*="pageloader"]
+[class*="title"] {
+    max-width: 100vw;
+    width: fit-content;
+    text-align: center;
+    white-space: normal;
+}
+
+[class*="fit-page"] {
+    max-width: 100vw;
+    max-height: 100vh;
+}
```

Comment:
Codacy found an issue: [Unexpected missing end-of-source newline (no-missing-end-of-source-newline)](https://app.codacy.com/gh/Project-Lavinia/Lavinia-client/pullRequest?prid=5991602)